# request
float64[] velocities
---
# response
bool done
